WiFi101WebThingAdapter.h

```diff
@@ -308,30 +308,8 @@ class WebThingAdapter {
             handleError();
           }
           return;
-        } else if (uri == deviceBase + "/properties") {
-          if (method == HTTP_GET || method == HTTP_OPTIONS) {
-            handleThingPropertiesGet(device->firstProperty);
-          } else {
-            handleError();
-          }
-          return;
-        } else if (uri == deviceBase + "/actions") {
-          if (method == HTTP_GET || method == HTTP_OPTIONS) {
-            handleThingActionsGet(device);
-          } else if (method == HTTP_POST) {
-            handleThingActionsPost(device);
-          } else {
-            handleError();
-          }
-          return;
-        } else if (uri == deviceBase + "/events") {
-          if (method == HTTP_GET || method == HTTP_OPTIONS) {
-            handleThingEventsGet(device);
-          } else {
-            handleError();
-          }
-          return;
-        } else {
```

Comment:
I suggest at least the /properties and /actions endpoints should be kept.

Reply:
Let me test it. If it doesn't break anything, I am fine with it.